qa/rpc-tests/addressindex.py

```diff
@@ -338,7 +338,7 @@ def check_balance(node_index, address, expected_balance, expected_received=None)
         assert_equal(deltas_info['end']['hash'], block_hash)
 
         # Test getaddressutxos by comparing results with deltas
-        utxos = self.nodes[1].getaddressutxos(addr1)
+        utxos = self.nodes[3].getaddressutxos(addr1)
```

Comment:
Node[3] is enabled for `-lightwalletd` only (not `-addressindex` like the other nodes), see the comment on line 60. So this change verifies that this RPC is available with only `-lightwalletd`. (This test fails without the main changes in this PR.)